src/components/DialPad.js

```diff
@@ -192,7 +192,8 @@ export default function DialPad() {
     if (phone.isInitialized()) {
       setCallStatus('Calling');
       setWebRtcStatus('Ringing');
-      setActiveCall(phone.call(`+${destNumber}`));
+      let extraHeaders = [`User-to-User:eyJhbGciOiJIUzI1NiJ9.WyJoaSJd.-znkjYyCkgz4djmHUPSXl9YrJ6Nix_XvmlwKGFh5ERM;encoding=jwt;aGVsbG8gd29ybGQ;encoding=base64`];
+      setActiveCall(phone.call(`+${phoneNumber}`, extraHeaders));
```

Comment:
```suggestion
      setActiveCall(phone.call(`+${destNumber}`, extraHeaders));
```

I think this should still be `destNumber` correct?